xstate/state_node.py

```diff
@@ -22,6 +22,12 @@ class StateNode:
     id: str
     key: str
     states: Dict[str, StateNode]
+    
+    def get_entries(self, entry_action):
```

Comment:
Should this be something like `get_actions` since it applies to exit actions too?

Reply:
Oh indeed, I noticed this just after sending the PR, but didn't have enough time on m'y break. 
I'll fix this quickly.

Thank you !